battleground_viewer/src/shaders/bloom_effect.frag

```diff
@@ -0,0 +1,36 @@
+uniform sampler2D emissive_buffer;
+
+in vec2 uvs;
+layout (location = 0) out vec4 color;
+
+const int size = 3;
+const float separation = 1.0;
+const float amount = 0.75;
+const float boost = 1.1;
```

Comment:
Absolutely! If you want to do so, change the declaration to `uniform` instead of `const`, and add the required `program.use_uniform(...);` calls to the effect application.

Reply:
Neat, good to know :+1: